website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
 title: Developing
 ---
 
-Developers are the demand side of The Graph ecosystem. Developers build subgraphs and publish them to The Graph Network. Then, they query live subgraphs with GraphQL in order to power their applications.
+Learn developer fundamentals and review the basics of a subgraph’s lifecycle. To start coding right away and engage The Graph, go to the following: [Developer Quick Start](/quick-start/)
 
-## Subgraph Lifecycle
+## Overview
 
-Subgraphs deployed to the network have a defined lifecycle.
+As a developer, you need important data to build and power your dapps. Querying and indexing that blockchain data can be challenging, but The Graph provides a solution to this issue.
 
-### Build locally
+On The Graph, you can:
 
-As with all subgraph development, it starts with local development and testing. Developers can use the same local setup whether they are building for The Graph Network or a local Graph Node, leveraging `graph-cli` and `graph-ts` to build their subgraph. Developers are encouraged to use tools such as [Matchstick](https://github.com/LimeChain/matchstick) for unit testing to improve the robustness of their subgraphs.
+1.  Create, deploy and publish subgraphs to The Graph using **Graph CLI** and [Subgraph Studio](/https://thegraph.com/studio/).
+2.  Use Graph node via **GraphQL** to query live subgraphs and power dapps.
 
-> There are certain constraints on The Graph Network, in terms of feature and network support. Only subgraphs on [supported networks](/developing/supported-networks) will earn indexing rewards, and subgraphs which fetch data from IPFS are also not eligible.
+### What is GraphQL and Graph CLI?
 
-### Deploy to Subgraph Studio
+- **GraphCLI** is a command line interface tool for building and deploying to The Graph.
+- **GraphQL** is the query language use for APIs and a runtime for executing those queries with your existing data. The Graph uses GraphQL to query subgraphs.
 
-Once defined, the subgraph can be built and deployed to [Subgraph Studio](/deploying/subgraph-studio-faqs/). Subgraph Studio is a sandbox environment which will index the deployed subgraph and make it available for rate-limited development and testing. This gives developers an opportunity to verify that their subgraph does not encounter any indexing errors, and works as expected.
+### Developer Capabilities
 
-### Publish to the Network
+- Engage with the entire subgraph lifecycle on the network, including publishing, querying and iterating on your custom subgraphs.
+- Specialize in subgraph development, building open APIs for others to utilize and build on.
+- Focus on application development by querying subgraphs deployed by other developers.
```

Comment:
The verbs here sound a bit marketing-y haha, and the last two bullet points sound a bit mutually exclusive, while it's totally possible to develop dapps _and_ subgraphs. Here's a very blunt suggestion:

```suggestion
- Query subgraphs built by other developers in their own dapps.
- Build their own subgraphs for their custom data needs (which can then be used by others to build on).
- Deploy, publish, and signal on their subgraphs.
```